cornflow-server/cornflow/endpoints/execution.py

```diff
@@ -306,6 +307,41 @@ def _raise_af_error(execution, error, state=EXEC_STATE_UNKNOWN):
         execution.update_state(state)
         return execution, 200
 
+    @doc(description="Change status of an execution", tags=["Executions"])
+    @authenticate(auth_class=Auth())
+    @use_kwargs(ExecutionStatusEndpointResponse)
+    @marshal_with(ExecutionStatusEndpointResponse)
```

Comment:
This schemas are already created, do they really expect how the data comes here?

The use_kwargs schema should only expect a status field that has to be an integer value.

And now that I think about it we do not need a marshal_with decorator or response schema. See the other comment